src/py/aspen/app/views/auspice.py

```diff
@@ -27,16 +27,21 @@ def auspice_view(phylo_tree_id: int):
         if not phylo_tree:
             raise RecoverableError(f"Phylo Tree {phylo_tree_id} not found.")
 
-        s3_client = boto3.resource(
+        s3_resource = boto3.resource(
             "s3",
             endpoint_url=os.getenv("BOTO_ENDPOINT_URL") or None,
             config=boto3.session.Config(signature_version="s3v4"),
         )
 
-        s3_object = s3_client.Object(phylo_tree.s3_bucket, phylo_tree.s3_key)
-        content = s3_object.get()["Body"].read().decode("utf-8")
+        s3_client = s3_resource.meta.client
 
-        response = make_response(content)
+        presigned_url = s3_client.generate_presigned_url(
+            "get_object",
+            Params={"Bucket": phylo_tree.s3_bucket, "Key": phylo_tree.s3_key},
+            ExpiresIn=3600,
+        )
+
+        response = make_response(json.dumps({"url": presigned_url}))
         response.headers["Content-Type"] = "text/json"
-        response.headers["Content-Disposition"] = "attachment; filename=auspice.json"
+
         return response
```

Comment:
do this instead:
```
return jsonify({"url": presigned_url})
```

Reply:
ok i added this and it works well (tested on remote dev stack too)

Reply:
once i verify that GHA passes i'll merge this in